<launch>
<!--    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lab6)/rviz/config.rviz" />-->
    <node name="stage" pkg="stage_ros" type="stageros" args="$(find lab6)/world/playground.world"/>
    <node name="perception" pkg="lab6" type="perception.py" output="screen"/>
    <node name="bug2" pkg="lab6" type="bug2.py" output="screen">
        <rosparam param="start_pos">[-7.0, 0.0]</rosparam>
        <rosparam param="goal_pos">[4.36, 9]</rosparam>
    </node>
</launch>
